tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "claz"
+weight = 4
+template = "doc.html"
++++
+
+This document outlines some advanced techniques for interacting with Azimuth contracts from Urbit itself. It's strongly discouraged to do this unless you're very confident as Azimuth transactions will cost Etherium to complete. However, until you upload a signed transaction to the block chain no transaction will take place so it's safe to explore these tools. It will also be very helpful to have at least a rudimentary understanding of Hoon in order to read and modify parts of the code.
+
+First you will want to boot a fakezod. You can do this with your live running ship but at this time this will need you to modify the code so it's best to do this on a ship you can make mistakes on. If you don't know how to boot a fakezod see this guide for instructions.
+
+Be sure to have mounted your home desk so you can easily edit files on it.
+The first edit that will probably need to be made is in [`app/claz.hoon`](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/claz.hoon#L14). The listed url will need to be changed to 
+
+https://mainnet.infura.io/v3/2599df54929b47099bda360958d75aaf
+
+or another valid endpoint.
+
+The second change that will potentially be needed is to modify the [gas limit](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/claz.hoon#L179). This number here is the maximum number of gas units that will be used. In future this will be configurable but for now you may need to change this manually. For a guide to what to set it to you can use the [constants in bridge](https://github.com/urbit/bridge/blob/master/src/lib/constants.js#L23) as a guide based on what action you are going to be preforming. You will need at least the maximum gas units multiplied by the gwei you're going to set later in your etherium account as it may cost that much though is likely to be far less.
+
+Once you have made these changes be sure to run `|commit %home` to get them into your ship.
+
+Next run `|start %claz` after which you'll be able to generate transactions.
+
+```
+:claz [%generate %/transaction/eth-txs %mainnet author %single %spawn ~ship to]
+```
+
+Here we have an example of generating a transaction to spawn a ship. Something to note is that for this to be successful you'll need to actually have the right to spawn the ship in question. Let's break down the command.
+
+
+`%generate` is how all commands to `claz` are going to start.
+
+`%/transaction/eth-txs` is simply the path for the output file
+
+`%mainnet` specifies which network this is going to sent out on.
+
+`author` here is an etherium address that is issuing the transaction. Remember that it will need to be written all in lower case, dotted notation, e.g.  0x3b17.d097.d9dd.711e.4ef8.517a.bbf1.8b2b.a643.81fe
```

Comment:
```suggestion
`author` here is an etherium address that is issuing the transaction. It will need to be written in `@ux` notation, e.g. `0x3b17.d097.d9dd.711e.4ef8.517a.bbf1.8b2b.a643.81fe`.
```